packages/fluent-theme/src/components/sendbox/Sendbox.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { type MouseEventHandler, useCallback, useRef, useState } from 'react';
+import cx from 'classnames';
+import { hooks } from 'botframework-webchat-api';
+import type { DirectLineCardAction, WebChatActivity } from 'botframework-webchat-core';
+import { SendIcon } from '../../icons/SendIcon';
+import { ToolbarButton, ToolbarSeparator, Toolbar } from './Toolbar';
+import { SuggestedActions } from './SuggestedActions';
+import { AddAttachmentButton, AttachmentDropzone, Attachments } from './Attachments';
+import { TextArea } from './TextArea';
+import { DialpadIcon } from '../../icons/DialpadIcon';
+import { useStyles } from '../../styles';
+
+const { useLocalizer } = hooks;
+
+const styles = {
+  webchat__sendbox__root: {
```

Comment:
This should be block if I am right.

```suggestion
  webchat__sendbox: {
```